padd.sh

```diff
@@ -257,6 +260,8 @@ GetSystemInformation() {
     temp_heatmap=${magenta_text}
   elif [ ${cpu} -gt 60000 ]; then
     temp_heatmap=${blue_text}
+  elif [ ${cpu} == 0]; then
```

Comment:
dschaper commented 2 minutes ago:
Try putting a space between the 0 and the ] on line 263.

Reply:
This worked for me